confluent-common-docker.yaml

```diff
@@ -50,12 +52,7 @@ subpackages:
       - uses: strip
 
 update:
-  enabled: true
-  version-transform:
-    - match: "-"
-      replace: "."
-  release-monitor:
-    identifier: 371725
+  enabled: false # Upstream repo doesn't provide any jars/poms past 7.6.0: https://packages.confluent.io/maven/io/confluent/common/
```

Comment:
I'm thinking this should be enabled and use the git update check. A tag filter that looks for 7.6 only, with a note to remove the filter once 7.7 jars are public would ensure we get updates while that branch is still being maintained by confluent.

Reply:
Right, since we're blocked from checking for releases, this won't work.